libvgm.mak

```diff
@@ -7,6 +7,6 @@ CFLAGS += -DAUDDRV_WINMM -DAUDDRV_DSOUND -DAUDDRV_XAUD2
 LDFLAGS_LIBVGM += -Wl,-Bstatic -lvgm-audio -lvgm-emu -lvgm-utils -Wl,-Bdynamic
 LDFLAGS_LIBVGM += -ldsound -luuid -lwinmm -lole32
 else
-CFLAGS += `pkg-config --with-path=/usr/local/lib/pkgconfig --cflags vgm-audio`
-LDFLAGS_LIBVGM += -Wl,-rpath,/usr/local/lib `pkg-config --with-path=/usr/local/lib/pkgconfig --libs vgm-audio vgm-emu`
+CFLAGS += `pkg-config --cflags vgm-audio`
```

Comment:
The "--with-path" is a parameter which is only supported in "pkgconf", an alternate implementation of pkg-config that is used by Arch Linux. Ideally we'd be able autodetect which version/implementation of pkg-config that is used to decide whether this should be specified or not.